src/geocat/viz/util.py

```diff
@@ -1248,22 +1441,59 @@ def set_vector_density(data: xr.DataArray,
         return ds
 
 
-def get_skewt_vars(p: Quantity, tc: Quantity, tdc: Quantity,
-                   pro: Quantity) -> str:
+def get_skewt_vars(pressure: Quantity = None,
+                   temperature: Quantity = None,
+                   dewpoint: Quantity = None,
+                   profile: Quantity = None,
+                   p: Quantity = None,
+                   tc: Quantity = None,
+                   tdc: Quantity = None,
+                   pro: Quantity = None) -> str:
     """This function processes the dataset values and returns a string element
     which can be used as a subtitle to replicate the styles of NCL Skew-T
     Diagrams.
 
     Parameters
     ----------
+    pressure : :class:`pint.Quantity`
+        Pressure level input from dataset. Renamed from deprecated kwarg `p`.
+    temperature : :class:`pint.Quantity`
+        Temperature for parcel from dataset. Renamed from deprecated kwarg `tc`.
+    dewpoint : :class:`pint.Quantity`
+        Dew point temperature for parcel from dataset. Renamed from deprecated kwarg `tdc`.
+    profile : :class:`pint.Quantity`
+        Parcel profile temperature converted to degC. Renamed from deprecated kwarg `pro`.
     p : :class:`pint.Quantity`
-        Pressure level input from dataset
+        Pressure level input from dataset.
+
+        .. deprecated:: 2023.06.0
+            In an effort to refactor the codebase to follow naming conventions,
+            keyword arguments have been renamed to more meaningful values.
+            ``p`` parameter has been deprecated in favor of ``pressure`.
+
     tc : :class:`pint.Quantity`
-        Temperature for parcel from dataset
+        Temperature for parcel from dataset.
+
+        .. deprecated:: 2023.06.0
+            In an effort to refactor the codebase to follow naming conventions,
+            keyword arguments have been renamed to more meaningful values.
+            ``tc`` parameter has been deprecated in favor of ``temperature`.
```

Comment:
These changes look great now!  It's not every Friday you can say that deprecation warnings bring you joy 😆 .

Somehow the last ` got dropped on each of the new parameters in the deprecation notes though (temperature, dewpoint, and profile) so it's rendering a little funky there.

Reply:
Oh gosh. Thanks for catching everything